pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -239,6 +239,9 @@ type PagerDutyConfig struct {
 	// HTTP client configuration.
 	// +optional
 	HTTPConfig *HTTPConfig `json:"httpConfig,omitempty"`
+	//Unique location of the affected system.
+	//+optional
+	Source string `yaml:"source,omitempty" json:"source,omitempty"`
```

Comment:
Since its a optional field it should be a pointer type
```suggestion
	Source *string `yaml:"source,omitempty" json:"source,omitempty"`
```

Reply:
Ref https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required